third-party/victoria-metrics-operator/scripts/download-crd.ts

```diff
@@ -1,28 +1,23 @@
 /* eslint-disable node/no-unpublished-import */
-import JSZip from "jszip";
 import { createWriteStream } from "fs";
 import { mkdir } from "fs/promises";
 import { dirname, join } from "path";
 import streamToPromise from "stream-to-promise";
-import assert from "assert";
 import fetch from "make-fetch-happen";
 import findCacheDir from "find-cache-dir";
 
-const VERSION = "0.37.1";
+const VERSION = "0.49.1";
 
 const outputPath = join(__dirname, "../crds/crd.yaml");
 
 (async () => {
   const res = await fetch(
-    `https://github.com/VictoriaMetrics/operator/releases/download/v${VERSION}/bundle_crd.zip`,
+    `https://github.com/VictoriaMetrics/operator/releases/download/v${VERSION}/crd.yaml`,
```

Comment:
It seems that this script could be deprecated now. You can move this URL to `crd-generate.input` in `package.json`.

Reply:
adjusted in https://github.com/tommy351/kubernetes-models-ts/pull/215/commits/bcd9939a902c9556bd1a8fd2b52a2e40b7503157